spec/features/course/assessment/submission/autograded_spec.rb

```diff
@@ -82,6 +82,16 @@
         expect(page).to have_selector('h3', text: mrq_questions.second.display_title)
       end
 
+      scenario 'I can directly go to next question if the assessment is skippable' do
+        extra_mrq_question
+        assessment.update_column(:skippable, true)
```

Comment:
I didn't do that cause it will require us to recreate everything ( submission + questions )...